src/components/LoginPage/Slide1.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import imageBrain from 'components/LoginPage/images/slide-1-brain.svg';
+import { Img } from 'components/LoginPage/slideComponents';
+
+const Slide = styled.section`
+  padding-bottom: 10%;
+  text-align: center;
+  Img {
+    width: 74%;
+  }
+`;
+const AppName = styled.header`
+  display: block;
```

Comment:
`<header>` has `display: block` by default.

Reply:
Done in commit [5c474d6](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/5c474d679beb09cdf8eb5db31e9ce6cf16a6d808)